packages/providers/src/lib/emailjs/emailjs.provider.ts

```diff
@@ -6,9 +6,8 @@ import {
   ICheckIntegrationResponse,
   CheckIntegrationResponseEnum,
 } from '@novu/stateless';
-import { Message, SMTPClient, MessageAttachment } from 'emailjs';
 import { IEmailJsConfig } from './emailjs.config';
-import { MessageHeaders } from 'emailjs/smtp/message';
+import type { Message, SMTPClient, MessageAttachment } from 'emailjs';
```

Comment:
This provider shipped ESM only and wasn't compatible with SWC, so we shift to use a runtime require.